frontend/src/lib/components/LemonRow/LemonRow.scss

```diff
@@ -51,6 +51,17 @@
     &.LemonRow--disabled {
         opacity: var(--opacity-disabled);
     }
+
+    &.LemonRow--large {
+        padding: 1rem 1rem;
+        font-size: 1rem;
+    }
+
+    &.LemonRow--small {
+        padding: 0.25rem 0.5rem;
+        font-size: 0.75rem;
+        min-height: 1.5rem;
+    }
```

Comment:
Note that symbolic rows (icon-only, `.LemonRow--symbolic`) need height AND width to be set firmly, otherwise they are a bit out of whack (icon-only buttons should be square) like so:
<img width="60" alt="Unfold" src="https://user-images.githubusercontent.com/4550621/166675008-82cb035b-bc48-4217-9d84-0c92af456590.png">
You can see `.LemonRow--compact` for reference.

Reply:
This icon should also use text muted or muted-alt for it's color. These are #5F5F5F and #747EA1.